tests/test_fixtures/rule_based_profiler/quentin_user_workflow_verbose_profiler_config.yml

```diff
@@ -13,33 +13,17 @@ variables:
   num_bootstrap_samples: 9139
   bootstrap_random_seed: 43792
 
-  # BatchRequest yielding thirty five (35) batches (January, 2018 -- November, 2020 trip data)
-  jan_2018_thru_nov_2020_monthly_tripdata_batch_request:
-    datasource_name: taxi_pandas
-    data_connector_name: monthly
-    data_asset_name: my_reports
-    data_connector_query:
-      index: ":-1"
-
   false_positive_rate: 5.0e-2
 
 rules:
   column_quantiles_rule:
     domain_builder:
       class_name: ColumnDomainBuilder
-      # BatchRequest yielding exactly one batch (November, 2020 trip data)
```

Comment:
Same thought here regarding preserving the knowledge.

Reply:
@Shinnnyshinshin I did just that (hopefully in sufficient number of places).